eclass/netsurf.eclass

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# @ECLASS: netsurf.eclass
+# @MAINTAINER:
+# maintainer-needed@gentoo.org
+# @SUPPORTED_EAPIS: 5 6 7
+# @BLURB: Handle buildsystem of www.netsurf-browser.org components
+# @DESCRIPTION:
+# Handle settings build environment for netsurf build system
+#
+# Supports PATCHES and DOCS as in base.eclass
+
+if [[ -z ${_NETSURF_ECLASS} ]]; then
+_NETSURF_ECLASS=1
+
+inherit toolchain-funcs
+
+# @FUNCTION: netsurf_define_makeconf
+# @DESCRIPTION:
+# This function sets NETSURF_MAKECONF as needed by netsurf build system
+netsurf_define_makeconf() {
```

Comment:
I think this needs a `@USAGE` per https://devmanual.gentoo.org/eclass-writing/index.html

mgorny has separated out the code that is used to turn these comments into man pages, so you can check that the output looks reasonable: https://github.com/mgorny/eclass-to-manpage